src/system/kernel_linux/Team.cpp

```diff
@@ -4,9 +4,26 @@
  */
 
 #include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <dirent.h>
+#include <list>
+
 
 #include "syscalls.h"
 
+using namespace std;
```

Comment:
Please don't use "using" unless it's really necessary :)

Reply:
it's really necessary, since if i don't add it, the compile fails.